src/js/files/components/UploadItem.js

```diff
@@ -26,16 +27,51 @@ const UploadItemTitle = styled.div`
     }
     span:last-child {
         margin-left: auto;
+        color: ${props => (props.failed ? getColor({ theme: props.theme, color: "red" }) : "inherit")};
+        font-size: ${props => (props.failed ? getFontSize("sm") : "inherit")};
+        font-style: ${props => (props.failed ? "italic" : "normal")};
+    }
+    i.fa-times {
+        color: ${props => getColor({ theme: props.theme, color: "red" })};
+        font-size: 20px;
+    }
+    i.fa-trash {
+        margin-left: 5px;
+        font-size: 14px;
     }
 `;
 
-export const UploadItem = ({ name, progress, size }) => (
-    <StyledUploadItem>
-        <AffixedProgressBar now={progress} />
-        <UploadItemTitle>
-            {progress === 100 ? <Loader size="14px" /> : <Icon name="upload" />}
-            <span>{name}</span>
-            <span>{byteSize(size)}</span>
-        </UploadItemTitle>
-    </StyledUploadItem>
-);
+export const UploadItem = ({ name, progress, size, failed, localId, onRemove }) => {
+    let [now, progressColor] = [progress, "blue"];
```

Comment:
Just define one variable per line. I am surprised ESLint is not complaining about this.

Actually, I prefer unary conditionals in the `return` statement for simple value switching like this.
```js
<AffixedProgressBar now={failed ? 100 : progress} color={failed ? "red" : "blue"} />
```

Reply:
Changed.